src/store/flow/flow.ts

```diff
@@ -82,7 +87,7 @@ export const mutations: MutationTree<IFlowsState> = {
   flow_setFirstBlockId(state, {flowId, blockId}) {
     const flow: IFlow = findFlowWith(flowId, state as unknown as IContext)
     const block: IBlock = findBlockWith(blockId, flow)  // @throws ValidationException when block absent
-    flow.firstBlockId = block.uuid
+    Vue.set(flow, 'firstBlockId', block.uuid)
```

Comment:
@safydy-r We'll want to always assume these objects are in flow-spec so that we needn't do this type of safety-netting throughout the codebase. 

Would you be willing to share what scenario we didn't have an observable as `flow.firstBlockId`?